contracts/HtsSystemContractJson.sol

```diff
@@ -463,6 +463,17 @@ contract HtsSystemContractJson is HtsSystemContract {
         return slot;
     }
 
+    function __nftInfoSlot(uint32 serialId) internal override virtual returns (bytes32) {
+        bytes32 slot = super.__nftInfoSlot(serialId);
+        if (_shouldFetch(slot)) {
+            string memory metadata = mirrorNode().getNftMetadata(address(this), serialId);
+            string memory createdTimestamp = mirrorNode().getNftCreatedTimestamp(address(this), serialId);
```

Comment:
we should unify these two calls to the MN into one, maybe something like `getNftMetadataAndCreatedTimestamp`, and extract both fields.